File: app/src/main/java/com/example/rodrigo/desafiomobile/network/service/GamesService.java
package com.example.rodrigo.desafiomobile.network.service;

import com.example.rodrigo.desafiomobile.entity.GamesListEntity;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

/**
 * metodo retrofit  para acessar url do json
 * lista de jogos do json
 */
public interface GamesService {
    @GET("s/1b7jlwii7jfvuh0/games")
    Call<GamesListEntity> getGames();

}

File: app/src/main/java/com/example/rodrigo/desafiomobile/network/service/GamesService.kt
package com.example.rodrigo.desafiomobile.network.service

import com.example.rodrigo.desafiomobile.entity.GamesListEntity
import io.reactivex.Observable
import retrofit2.http.GET

interface GamesService {
    @GET("s/1b7jlwii7jfvuh0/games")
    fun listGames(): Observable<GamesListEntity>
}